oep-template.rst

```diff
@@ -0,0 +1,85 @@
+=================================
+OEP-1: OEP Purpose and Guidelines
+=================================
+
++---------------+-------------------------------------------+
+| OEP           | <link to OEP>                             |
++---------------+-------------------------------------------+
+| Title         | <OEP title>                               |
++---------------+-------------------------------------------+
+| Version       | <version string>                          |
++---------------+-------------------------------------------+
+| Last-Modified | <date string>                             |
++---------------+-------------------------------------------+
+| Author        | <list of authors' real names and          |
+|               | optionally, email addrs>                  |
++---------------+-------------------------------------------+
+| Arbiter       | <OEP czar's real name>                    |
++---------------+-------------------------------------------+
+| Status        | <Draft | Accepted | Deferred |            |
+|               | Rejected | Withdrawn | Final |            |
+|               | Replaced>                                 |
++---------------+-------------------------------------------+
+| Type          | <Architecture | Best Practice |           |
+|               | Process>                                  |
++---------------+-------------------------------------------+
+|  Created      | <date created on, in dd-mmm-yyyy format>  |
++---------------+-------------------------------------------+
+| `Replaces`    | <OEP number>                              |
++---------------+-------------------------------------------+
+| `Replaced-By` | <OEP number>                              |
++---------------+-------------------------------------------+
+
+Abstract
+========
+a short (~200 word) description of the technical issue being addressed.
```

Comment:
I think it's easier to read the .rst files if there is a blank line after the underline line:
```
Abstract
========

A short (~200 word) description...
```